src/server/pfs/fuse/fuse_test.go

```diff
@@ -279,7 +279,7 @@ func TestRepoOpts(t *testing.T) {
 			},
 		},
 		RepoOptions: map[string]*RepoOptions{
-			"repo1": {},
+			"repo1": {Repo: "repo1"},
```

Comment:
I don't actually know the fuse library's API that well, but if RepoOptions is part of the client's user-facing API, we may be obligated by semver to infer a default value for `Repo` if it's unset (in case an existing user is calling our client with `Repo` unset; they should be able to upgrade without it breaking)

Reply:
I think it's just consumed by the CLI for `pachctl mount` which has been updated.